pkg/login/brute_force_login_protection.go

```diff
@@ -44,5 +44,7 @@ var saveInvalidLoginAttempt = func(query *m.LoginUserQuery) {
 		IpAddress: query.IpAddress,
 	}
 
-	bus.Dispatch(&loginAttemptCommand)
+	if err := bus.Dispatch(&loginAttemptCommand); err != nil {
+		// TODO: Deal with error
```

Comment:
We want to log this somehow. `saveInvalidLoginAttempt` can return an error and caller can log it. Caller, in this case auth.go, May create a `var loginLogger = log.New("login")` and use that in auth.go. Why name the logger `login`? Because there's one named login.ldap so trying to follow the same convention.